packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/e2e/pageObject.ts

```diff
@@ -0,0 +1,295 @@
+import type {Locator, Page, Request} from '@playwright/test';
+import {
+  isUaClickEvent,
+  isUaCustomEvent,
+} from '../../../../../../playwright/utils/requests';
+
+const minimumCitationTooltipDisplayDurationMs = 1500;
+
+export class GeneratedAnswerObject {
+  constructor(
+    public page: Page,
+    public streamId: string
+  ) {
+    this.page = page;
+    this.streamId = streamId;
+  }
+
+  get likeButton(): Locator {
+    return this.page.locator('[data-cy="feedback__like-button"]');
+  }
+
+  get dislikeButton(): Locator {
+    return this.page.locator('[data-cy="feedback__dislike-button"]');
+  }
+
+  get copyToClipboardButton(): Locator {
+    return this.page.locator(
+      '[data-test="generated-answer__copy-to-clipboard"]'
+    );
+  }
+
+  get toggleButton(): Locator {
+    return this.page.locator('[data-test="generated-answer__toggle-button"]');
+  }
+
+  feedbackOption(value: string, index: number): Locator {
+    return this.page
+      .locator('.feedback-modal-qna__question input', {
```

Comment:
This can be made better with something like this I'm sure:

`page.getByRole('input').filter({hasText : ???})`